environment.yml

```diff
@@ -0,0 +1,57 @@
+name: freqtrade
+channels:
+  - defaults
+dependencies:
+  # Required for app
+  - python>=3.7
+  - pip
+  - wheel
+  - numpy=1.16.4
+  - pandas=0.24.2
+  - scipy=1.3.0
+  - SQLAlchemy=1.3.5
+  - scikit-learn=0.21.2
+  - arrow=0.14.2
+  - requests=2.22.0
+  - urllib3=1.24.2
+  - wrapt=1.11.2
+  - joblib=0.13.2
+  - jsonschema=3.0.1
+  - tabulate=0.8.3
+  - python-rapidjson=0.7.2
+  - filelock=3.0.12
+  - flask=1.0.3
+  - python-dotenv
+  - cachetools=3.1.1
+  - scikit-optimize=0.5.2
+  - python-telegram-bot=11.1.0
+  # Optional for plotting
+  - plotly=3.10.0
+  # Optional for development
+  - flake8=3.7.7
+  - pytest=5.0.0
+  - pytest-mock=1.10.4
+  - pytest-asyncio=0.10.0
+  - pytest-cov=2.7.1
```

Comment:
these pinned dependencies could become a problem.
we use pyup to update dependencies, which cannot update a environment.yml file (https://github.com/pyupio/pyup/issues/322).

manually updating these will be problematic i think.

Reply:
Agreed. Conda does a good job of working out dependencies and I don't typically include any versions. But there was a conflict it missed that caused the install to fail probably because there are so many that have to be installed with pip. I'll try to narrow that down. If it's just one or two packages, `environment.yml` can likely stay the same until those few packages change.

Reply:
I removed the versions and it works.  I think pinning python fixed it.